docs/source/contributing/dev-contributions/contrib_guide.rst

```diff
@@ -121,6 +121,7 @@ including partial or in-progress work, is appreciated.
 Contribution Workflow
 ---------------------
 .. image:: ../../_static/_images/contribution_workflow.png
+   :alt: 
```

Comment:
```suggestion
   :alt: A flow chart with 14 labeled boxes linked by arrows. The chart is uni-directional.  At each step, arrows point forward to one or more boxes and back to the previous box or boxes. Refer to the link above the image for full, long alt-text.
```

Reply:
File I'm working on: docs/source/contributing/dev-contributions/contrib_guide.rst

Link to [image in documentation](https://github.com/isabela-pf/jupyter/blob/9d0a04da8d7bdb4b85098e6960783f8691bfa275/docs/source/contributing/dev-contributions/contrib_guide.rst#contribution-workflow)
![image](https://user-images.githubusercontent.com/46167686/150649845-4de41167-5e52-4715-aea7-3f0aa1257f83.png)

Short alt-text for above image:
' A flow chart with 14 labeled boxes linked by arrows. The chart is uni-directional.  At each step, arrows point forward to one or more boxes and back to the previous box or boxes. Refer to the link above the image for full, long alt-text.'

Link to [long alt-text I wrote, hosted on a gist](https://gist.githubusercontent.com/MarsBarLee/2dc5e9c83adf0555d08277b2262302e5/raw/be6a0c46f58d7135a75df7747698cef676b9cdc0/Jupyter%2520Documentation:%2520Contribution%2520Workflow%2520Alt-Text).